README.md

```diff
@@ -72,61 +72,52 @@ These commands are available after installation within the `image-annotator/` di
 - `npm run storybook`: launch [React Storybook](https://getstorybook.io) for component development (accessible at [localhost:6006](http://localhost:6006) by default)
 
 ### backend
-The backend is running node, Postgres and an express based set of middlewares for the REST API we expose.
+The backend is running node, PostgreSQL and an express based set of middlewares for the REST API we expose. In order to develop the application, you must have an installation of PostgreSQL.
 
-The app must be pointed at a running postgres server. Default config will use the db name `image-annotator`, and a set of sensible connection defaults, but if you need to, you can configure all the connection parameters in the `.gitignored` `.env` file. For example:
+#### Setting up Postgres
+There are numerous ways to run PostgreSQL. Choose the one that is most familiar
+to you!
 
+**If you have Docker on your machine, do the following:**
 ```
-PGHOST=localhost
-PGPORT=5432
-PGNAME=image-annotator
-PGUSER=postgres
-PGPASS=*****
-# if specified, DATABASE_URL overides all of the above
-DATABASE_URL=
+docker run --name ia-pg -d -p 5432:5432 -i postgres
+docker exec -it ia-pq su postgres -c 'createdb image-annotator'
+npm migrate
+npm start
 ```
```

Comment:
Is it worth mentioning you can then stop and start the same container with `docker stop ia-pg` and `docker start ia-pg`. The start command may be especially useful to mention after restarting the host.